db-scripts/src/main/resources/migration.sql

```diff
@@ -720,3 +720,36 @@ CASE
     ELSE 1
 END;
 UPDATE `info` SET `DB_SCHEMA_VERSION`="2.11.0";
+
+##version 2.11.1
+CREATE TEMPORARY TABLE IF NOT EXISTS
+    fusion_studies ( INDEX(CANCER_STUDY_IDENTIFIER) )
+    ENGINE=MyISAM
```

Comment:
Does the engine on these temporary tables have to be `MyISAM`? I thought our defaults were `InnoDB` - please correct me if I'm wrong.

Reply:
Either should work. Only needs to index one field. MyISAM is lighter and faster than InnoDB.

Reply:
okay. will drop the engine reference.

Reply:
done.